doc/source/tutorial/optimize.rst

```diff
@@ -70,6 +74,31 @@ Another optimization algorithm that needs only function calls to find
 the minimum is *Powell*'s method available by setting ``method='powell'`` in
 :func:`minimize`.
 
+To demonstrate how to supply additional arguments to an objective function, 
+let us minimize the Rosenbrock function with an additional scaling factor `a` 
+and an offset `b`:
+
+.. math::
+
+    f\left(\mathbf{x}, a, b\right)=\sum_{i=1}^{N-1}a\left(x_{i+1}-x_{i}^{2}\right)^{2}+\left(1-x_{i}\right)^{2} + b.
+
+Again using the :func:`minimize` routine this can be solved by the following 
+code block for the exemplary parameters `a=0.5` and `b=1`. 
```

Comment:
```suggestion
code block for the example parameters `a=0.5` and `b=1`. 
```
"exemplary" may technically be valid, but I think most readers would interpret "exemplary" as meaning something like "deserving imitation because of excellence". Here, we mean the same as "sample" parameters, so I'd suggest "example" (where "example" is used as an attributive noun or adjective, [depending on who you ask](https://forum.wordreference.com/threads/example-as-an-adjective-example-sentence.1772545/#:~:text=Added%3A,about%20words%3A%20their%20...)).